meerk40t/core/node/mixins.py

```diff
@@ -3,6 +3,9 @@
 
 
 class Stroked(ABC):
+    def __init__(self, *args, **kwargs):
+        self.mkparam = None
```

Comment:
Better, but maybe you could have a non-stroked element node with functional parameters. This isn't really a requirement of stroked node.